src/main/java/org/dsa/iot/alarm/AbstractAlarmObject.java

```diff
@@ -117,9 +119,22 @@ public int childCount() {
      * Convenience for delete actions.
      */
     public void deleteSelf() {
+        ArrayList<UUID> toClose = new ArrayList<UUID>();
+        getOpenUUIDs(toClose);
```

Comment:
Why not create the list inside the `getOpenUUIDs()` method? Seems like it'd remove the parameter too.